### ISSUE TITLE
Remove message object for `chat.send`

### ISSUE THREAD

**Issue by nathunsmitty** [keybase/keybase-bot#96]
It feels a bit odd to me to have the `ChatMessage` type for use on the bot's `chat.send` method. For reference, `ChatMessage` looks like:

```javascript
/**
 * A chat message. The content goes in the `body` property!
 */
export type ChatMessage = {|
  body: string,
|}
```

I think it's simpler for users to just input the text (`body`) of their message directly and let the `chat.send` method handle the formatting of arguments for them. Ideally, usage of `chat.send` would look like:

```javascript
const channel = {name: 'kbot,' + bot.myInfo().username, public: false, topic_type: 'chat'}
bot.chat.send(channel, 'Hello kbot!').then(() => console.log('message sent!'))
```

@malgorithms @thebearjew thoughts? This would be a breaking change.

**Comment by malgorithms** [comment]
I'm ok either way. I wouldn't worry about breaking changes since I think no one is using our module yet, and our README has warnings about its instability. But we can do an appropriate version upgrade.

I think the question to ask for this change is whether `send` will take anything other than body texts in the future. For example, will attachment sending be something like this?

```
bot.chat.send(channel, {
   file: "me.jpg",
   caption: "Hot selfie"
}).then...

```

these aren't "options" they're what is being sent.

Or will sending an attachment be an entirely different function call? In which case your change makes sense.

Hmm. how about this: we just wait a bit until wde do attachments and then decide

**Comment by nathunsmitty** [comment]
Good point. I was thinking attachments would be a different method, since that's what we do in the command line

**Comment by nathunsmitty** [comment]
Given that we now have a separate attach method, @malgorithms what do you think about this? Both the Go and Python bots just accept strings instead of the message object.
